### ISSUE TITLE
Parameter choice for circular constraint optimizer

### ISSUE THREAD

**Issue by bgobbi** [TiffanyVlaar/ConstrainedNNtraining#1]
Hi 
I am trying to use the optimizer with circular constraints to train a network that is sensitive to the initial random weight initialization. I have integrated the cCoLAud Optimizer into my code and followed your implementation of the circleconstraint_FashionMNIST_MLP.py to initialize the optimizer.
I have tried using your default parameters and using bayesian optimization to choose good parameters for the optimizer. So far I have failed. With the default parameters the Network results in Infinity already in the first epoch. When using the bayesian optimization some parameters that work are found but the optimization gets stuck on both the training and validation loss at significantly higher loss values than are achieved with SGD or Adam.

Do you have any advise on how to choose the parameters for the cCoLAud Optimizer? I have seen that you also use a transformation for cgamma and dgamma. I do not completely understand all the parameters. Is there a description of their meaning?

**Comment by TiffanyVlaar** [comment]
Hi there, thank you for your interest! Hyperparameters for the provided experiments are given in the paper. The cgamma and dgamma parameters are defined in the code. The cCoLAud training scheme is described in Appendix B.4 of the paper (arXiv:2106.10704).
